src/hello-world/variables-and-values.md

```diff
@@ -18,6 +39,12 @@ The types have widths as follows:
 
 <details>
 
+Uncomment the `x = 20` to demonstrate that variables are immutable by default.
+Add the `mut` keyword to allow changes.
+
+Due to type inference the `i32` is optional. We will gradually show the types
+less and less as the course progresses.
```

Comment:
The `i32` fallback isn't really inferred from anything in the context here, but is the fallback that Rust's type inference uses for integral types when they're underconstrained after inference. A discussion of inference is worthwhile but should probably make this distinction and mention the f64 fallback for floats, especially as the two differ in byte size which may be surprising.